src/sections/dataset/dataset-citation/DatasetCitation.tsx

```diff
@@ -0,0 +1,69 @@
+import { Col, Icon, Row, Tooltip } from 'dataverse-design-system'
+import styles from './DatasetCitation.module.scss'
+import { useTranslation } from 'react-i18next'
+import { Citation, DatasetStatus } from '../../../dataset/domain/models/Dataset'
+
+interface DatasetCitationProps {
+  citation: Citation
+  status: DatasetStatus
+  version: string | null
+}
+interface CitationDatasetStatusProps {
+  status: DatasetStatus
+}
+
+function CitationDescription({ citation, status, version }: DatasetCitationProps) {
+  return (
+    <span>
+      {citation.citationText}, {citation.pidUrl}, {citation.publisher}
+      {version && ', V' + version.substring(0, 1)}
```

Comment:
`version.substring(0, 1) `this is taking only the first character of the string so if the version is 12.4.5 it would show V1 instead of V12. We need to take everything until the first dot